src/applications/representative-search/components/search/SearchResultsHeader.jsx

```diff
@@ -13,11 +13,6 @@ export const SearchResultsHeader = props => {
 
   const noResultsFound = !searchResults || !searchResults.length;
 
-  const sortOptions =
```

Comment:
I think this means we'll always use the org sort options? I'm unable to test locally right now.

Reply:
Talked offline and this is intentional for now